packages/main/test/specs/Input.spec.js

```diff
@@ -413,6 +413,35 @@ describe("Input general interaction", () => {
 			"The value is restored as ESC has been pressed.");
 	});
 
+	it("Input value should correspond to suggestion item when it is clicked", async () => {
+		await browser.url(`test/pages/Input.html`);
+
+		const suggestionsInput = await browser.$("#myInput").shadow$("input");
+
+		const staticAreaItemClassName = await browser.getStaticAreaItemClassName("#myInput");
+		const respPopover = await browser.$(`.${staticAreaItemClassName}`).shadow$("ui5-responsive-popover");
+
+		// paste a value completely corresponding to item text
+		await suggestionsInput.click();
```

Comment:
the click is redundant

Reply:
We are making sure that the input is focused in order to mimic the scenario. Also, this is present in all tests so I prefer to leave it.